### PR TITLE
[Improve] webui async function  improve

### PR DESCRIPTION
**Pull request by wolfboys**
[Improve] webui async function  improve

### CODE REVIEW

**Comment by wolfboys** [comment]
cc @wangsizhu0504 PTAL, thanks

File: streampark-console/streampark-console-webapp/src/views/flink/app/Add.vue [hunk 1]
```diff
@@ -122,7 +121,7 @@
       }
     } else {
       openConfDrawer(false);
-      setFieldsValue({ isSetConfig: false, configOverride: null });
+      await setFieldsValue({ isSetConfig: false, configOverride: null });
```
Comment (wangsizhu0504) [inline streampark-console/streampark-console-webapp/src/views/flink/app/Add.vue]: It seems that there is no need to add await here. First of all, setFieldValue is not an asynchronous function and the function is to assign a value and does not need to be carried out asynchronously. Secondly, there is no code logic behind setFieldValue, and there is no need to await here.

File: streampark-console/streampark-console-webapp/src/views/flink/variable/components/VariableDrawer.vue [hunk 2]
```diff
@@ -150,7 +150,7 @@
     async (data: Recordable) => {
       try {
         variableId.value = null;
-        resetFields();
+        await resetFields();
```
Comment (wangsizhu0504) [inline streampark-console/streampark-console-webapp/src/views/flink/variable/components/VariableDrawer.vue]: ResetFieldValue is not an asynchronous function. It is not recommended to add the await keyword. In fact, it is just an assignment function and does not need to be performed asynchronously

File: streampark-console/streampark-console-webapp/src/views/flink/app/EditStreamPark.vue [hunk 3]
```diff
@@ -305,7 +304,7 @@
       project: app.projectId,
       ...defaultFormValue,
     });
-    nextTick(() => {
+    await nextTick(() => {
```
Comment (wangsizhu0504) [inline streampark-console/streampark-console-webapp/src/views/flink/app/EditStreamPark.vue]: This is the content of editor settings. It should not affect the following logic, and it is not recommended to add await

**Review by wangsizhu0504 — DISMISSED** [review]
LGTM

File: streampark-console/streampark-console-webapp/src/views/flink/app/Add.vue [hunk 4]
```diff
@@ -227,7 +225,7 @@
     }
 
     let config = values.configOverride;
-    if (config != null && config !== undefined && config.trim() != '') {
+    if (config != null && config.trim() != '') {
```
Comment (wangsizhu0504) [inline streampark-console/streampark-console-webapp/src/views/flink/app/Add.vue:228]: if (config != null && config !== undefined && config.trim() != '') {

File: streampark-console/streampark-console-webapp/src/views/flink/app/Add.vue [hunk 5]
```diff
@@ -277,7 +275,7 @@
     const param = {};
     for (const k in params) {
       const v = params[k];
-      if (v != null && v !== undefined) {
+      if (v != null) {
```
Comment (wangsizhu0504) [inline streampark-console/streampark-console-webapp/src/views/flink/app/Add.vue:278]: if (v != null && v !== undefined) {

File: streampark-console/streampark-console-webapp/src/views/flink/app/EditStreamPark.vue [hunk 6]
```diff
@@ -357,7 +356,7 @@
       <template #args="{ model }">
         <ProgramArgs
           ref="programArgRef"
-          v-if="model.args != null && model.args != undefined"
+          v-if="model.args != null"
```
Comment (wangsizhu0504) [inline streampark-console/streampark-console-webapp/src/views/flink/app/EditStreamPark.vue:359]: v-if="model.args != null && model.args != undefined"

**Review by wangsizhu0504 — APPROVED** [review]
LGTM

**Pull request merged**